fluent-syntax/src/index.js

```diff
@@ -15,14 +15,22 @@ export function serialize(resource, opts) {
 }
 
 export function lineOffset(source, pos) {
-  // Substract 1 to get the offset.
+  // Subtract 1 to get the offset.
   return source.substring(0, pos).split('\n').length - 1;
 }
 
 export function columnOffset(source, pos) {
-  const lastLineBreak = source.lastIndexOf('\n', pos);
-  return lastLineBreak === -1
-    ? pos
-    // Substracting two offsets gives length; substract 1 to get the offset.
-    : pos - lastLineBreak - 1;
+  // Find the last line break starting backwards from the index just before
+  // pos.  This allows us to correctly handle ths case where the character at
+  // pos  is a line break as well.
+  const fromIndex = pos - 1;
```

Comment:
Without this, `columnOffset` was returning `0` when the character at `pos` was a new line.